src/components/Recipe/Item/Item.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { render, fireEvent, waitFor, screen } from '@testing-library/react'
+import { RenderAndHistory } from 'utils/test/types'
+import { createMemoryHistory, MemoryHistoryBuildOptions } from 'history'
+import { Router } from 'react-router-dom'
+
+import RecipeItem from './Item'
+import { Recipe } from '../../../data/recipes/types'
+
+const recipe = {
+  id: 1,
+  name: 'Chocolate cookies',
+  description: 'The best fudgy cookies you will ever try',
+  ingredients: [{ name: 'Chocolate' }, { name: 'Butter' }, { name: 'Flour' }],
+}
+
+interface Props {
+  recipe: Recipe;
+}
+
+function renderRecipeItem(props: Props): RenderAndHistory {
+  const history = createMemoryHistory()
+  const utils = render(
+    <Router history={history}>
+      <RecipeItem {...props} />
+    </Router>
+  )
+
+  return { ...utils, history }
+}
+
+describe('<RecipeItem>', () => {
+  it('should render a recipe', async () => {
+    renderRecipeItem({ recipe });
+  })
+
+  it('should render a recipe name', async () => {
+    const { getByText } = renderRecipeItem({ recipe });
```

Comment:
Prefer using `screen.getByText` (which I saw you're already using in some other files)

Reply:
Done, thanks